Makefile.work

```diff
@@ -185,6 +185,7 @@ SONIC_BUILD_INSTRUCTION :=  make \
                            SHUTDOWN_BGP_ON_START=$(SHUTDOWN_BGP_ON_START) \
                            INSTALL_KUBERNETES=$(INSTALL_KUBERNETES) \
                            KUBERNETES_VERSION=$(KUBERNETES_VERSION) \
+			   KUBERNETES_CNI_VERSION=$(KUBERNETES_CNI_VERSION) \
```

Comment:
tabs to spaces.

Reply:
Spacing seems correct in other views, but it looks off in the code snippet from PR review

My local file:
![image](https://user-images.githubusercontent.com/67024108/88315458-ef205880-ccdb-11ea-8100-b8a9c886bd18.png)
Clicking on 'View file' on GitHub: 
![image](https://user-images.githubusercontent.com/67024108/88315577-11b27180-ccdc-11ea-82fc-492a97c02188.png)

Reply:
Open in vi and search for tabs. (Type / followed by cntrl-v & cntrl-i to search for tabs)

Reply:
Thank you, it is fixed now